cellpack/autopack/ingredient/single_sphere.py

```diff
@@ -20,113 +20,144 @@ class SingleSphereIngr(Ingredient):
 
     def __init__(
         self,
-        Type="SingleSphere",
+        radius,
+        type="single_sphere",
```

Comment:
Could we merge `SingleSphere` and `MultiSphere` some day? and treat a single sphere as a multisphere with a single sphere in it.... ?